test/testlib/nuodb_database_utilities.go

```diff
@@ -151,6 +151,7 @@ type DatabaseInstallationStep func(t *testing.T, options *helm.Options, helmChar
 
 func StartDatabaseTemplate(t *testing.T, namespaceName string, adminPod string, options *helm.Options, installationStep DatabaseInstallationStep, awaitDatabase bool) (helmChartReleaseName string) {
 	randomSuffix := strings.ToLower(random.UniqueId())
+	randomSuffix = randomSuffix[:5]
```

Comment:
Can you add a comment saying why you're doing this?

I think this is totally fine. In fact, I'm not sure why we bother with creating unique names for namespaced resources, since the namespaces are generated to have unique names themselves. Seems like total overkill.

You can also just do the following:

```go
randomSuffix := strings.ToLower(random.UniqueId())[:5]
```